Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -657,18 +657,23 @@
   components:
   - type: SolutionContainerManager
     solutions:
-      hypospray:
+      injector: # Starlight edit
         maxVol: 10
   - type: RefillableSolution
-    solution: hypospray
+    solution: injector # Starlight edit
   - type: ExaminableSolution
-    solution: hypospray
+    solution: injector # Starlight edit
     heldOnly: true # Allow examination only when held in hand.
     exactVolume: true
-  - type: Hypospray
```

Comment:
Generally it's preferred to leave removed upstream code commented out rather than removing it, to leave context for merge conflict resolutions.